theano/gpuarray/extra_ops.py

```diff
@@ -40,7 +39,6 @@ def get_params(self, node):
 
     def make_node(self, x):
         assert x.type.dtype == 'float32', "Only float32 supported for GpuCumSum"
-        x = as_gpuarray_variable(x, infer_context_name(x))
```

Comment:
After having read the documentation, the context should be inferred from the inputs in general, not hardcoded in the Op itself (unless the Op has no GPU input, for instance GpuAllocEmpty or GpuEye).